src/emulator/auth/operations.ts

```diff
@@ -1755,6 +1750,27 @@ function generateJwt(
   return jwtStr;
 }
 
+function getAuthTime(user: UserInfo): Date {
+  if (user.lastLoginAt != null) {
```

Comment:
`!==`

Reply:
This is intentional since it can be `undefined`. TS won't let me do `!== null`